tests/trace/dsl/test_query.py

```diff
@@ -347,6 +347,26 @@ async def test_filter_for_none(
         check_index_type=False,
     )
 
+    sq = (
+        SpanQuery()
+        .select("name")
+        .where(
+            "parent_id is not None",
+        )
+    )
+    expected = pd.DataFrame(
+        {
+            "context.span_id": ["234", "345", "456", "567"],
+            "name": ["root span", "embedding span", "retriever span", "llm span"],
+        }
+    ).set_index("context.span_id")
+    async with db() as session:
+        actual = await session.run_sync(sq, project_name="abc")
+    assert_frame_equal(
+        actual.sort_index().sort_index(axis=1),
+        expected.sort_index().sort_index(axis=1),
+    )
+
```

Comment:
This doesn't match the name of the test case. Seems like a separate test to me.

Reply:
sorry, i thought i removed these, but somehow i didn't